MapboxMobileEvents/MMEEventsManager.m

```diff
@@ -157,6 +157,12 @@ - (void)flush {
         return;
     }
     
+    NSDictionary *flushEventAttributes = @{MMEEventKeyEvent: MMEEventTypeFlush,
+                                          MMEEventKeyCreated: [self.dateWrapper formattedDateStringForDate:[self.dateWrapper date]]};
+    MMEEvent *flushEvent = [MMEEvent pushEventWithAttributes:flushEventAttributes];
```

Comment:
With the changes in this PR so far, I think this should actually be `flushEventWithAttributes:`. But let's make the change to use the unified `debugEventWithAttributes`. As a debug event, the event name will be `MMEEventTypeLocalDebug` (that is done for us by `debugEventWithAttributes:`) and we could introduce a "debug event type" that we put in the attributes. In this case, that could be a string constant called  `MMEDebugEventTypeFlush` that equals "flush".